spec/requests/api/rest/admin/cdr/cdr_exports_spec.rb

```diff
@@ -191,8 +191,11 @@
           customer_auth_external_id_eq: 2_151_321,
           failed_resource_type_id_eq: 25,
           src_prefix_in_contains: '1111',
+          src_prefix_in_eq: '1111',
           dst_prefix_in_contains: '2222',
```

Comment:
lets add `dst_prefix_in_eq, src_prefix_out_eq, dst_prefix_out_eq` too

it will be useful